main.go

```diff
@@ -86,14 +91,16 @@ func showTimeDuration(durations map[string]time.Duration) {
 }
 
 // showErrorMessages shows table with all error messages gathered during action
-func showErrorMessages(action string, errorMessages map[string]error) error {
-	if len(errorMessages) > 0 {
+func showErrorMessages(action string, errorMessages map[string]ErrorMessage) error {
+	if hasErrorsToPrint(errorMessages) {
 		fmt.Println()
 		fmt.Printf("The following errors were encountered during %s:\n\n", action)
 		w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
-		_, _ = fmt.Fprintln(w, "STEP\tERROR\t")
-		for svc, err := range errorMessages {
-			_, _ = fmt.Fprintf(w, "%v\t%v\n", svc, err)
+		_, _ = fmt.Fprintln(w, "TYPE\tSTEP\tERROR\t")
+		for svc, error := range errorMessages {
```

Comment:
Two notes:
* Local variable `error` collides with builtin `error`!
* What does `svc` stands for? I don't know it. If you don't know it too, then I would rename it something obvious like `id`.

Reply:
svc indicates the step: `rhsm`, `insights-client` or `rhc` maybe I can change it to step instead.